project_event/i18n/fr.po

```diff
@@ -191,13 +190,13 @@ msgid "Activity:"
 msgstr "Activité:"
 
 #. module: project_event
-#: code:addons/project_event/models/project_project.py:182
+#: code:addons/project_event/models/project_project.py:184
 #, python-format
 msgid "Activity: "
 msgstr "Activité: "
 
 #. module: project_event
-#: code:addons/project_event/models/project_task.py:882
+#: code:addons/project_event/models/project_task.py:891
```

Comment:
Two entry for activity for two files. can we keep one entry referencing the two files